core/templates/dev/head/player/AnswerClassificationServiceSpec.js

```diff
@@ -34,6 +44,7 @@ describe('Answer classification service', function() {
         value: 'content'
       }],
       interaction: {
+        id: 'RuleTest',
```

Comment:
I believe this is now handled -- see the "should query the backend if no answer group matches" test below.